onnx_tf/handlers/backend/if.py

```diff
@@ -0,0 +1,70 @@
+import tensorflow as tf
+
+import onnx_tf
+from onnx.helper import make_opsetid
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("If")
+@tf_func(tf.cond)
+class If(BackendHandler):
+
+  @classmethod
+  def _common(cls, node, **kwargs):
+    cond = kwargs["tensor_dict"][node.inputs[0]]
+    then_branch = node.attrs["then_branch"]
+    else_branch = node.attrs["else_branch"]
+    current_opset = [make_opsetid(cls.DOMAIN, cls.VERSION)]
+
+    def true_fn():
+      input_values = {}
+      for i in then_branch.input:
+        input_values[i.name] = kwargs["tensor_dict"][i.name]
+      # get the required initializer constant node(s) for the then-branch graph
+      nodes_outputs = []
+      for n in then_branch.node:
+        for o_name in n.output:
+          nodes_outputs.append(o_name)
+      for n in then_branch.node:
+        for i_name in n.input:
+          # Need to get the initializer constant nodes from tensor_dict here
+          # because input from initializer will not be send in as inputs
+          # to the then-branch graph and those nodes are not in the
+          # then-branch graph
+          if i_name not in nodes_outputs and i_name not in input_values.keys():
+            input_values[i_name] = kwargs["tensor_dict"][i_name]
+      tensor_dict = onnx_tf.backend.onnx_graph_to_tensorflow_ops(
+          graph_def=then_branch, input_values=input_values, opset=current_opset)
+      return [tensor_dict[o.name] for o in then_branch.output]
+
+    def false_fn():
+      input_values = {}
+      for i in else_branch.input:
+        input_values[i.name] = kwargs["tensor_dict"][i.name]
+      # get the required initializer constant node(s) for the else-branch graph
+      nodes_outputs = []
+      for n in else_branch.node:
+        for o_name in n.output:
+          nodes_outputs.append(o_name)
+      for n in else_branch.node:
+        for i_name in n.input:
+          # Need to get the initializer constant nodes from tensor_dict here
+          # because input from initializer will not be send in as inputs
+          # to the else-branch graph and those nodes are not in the else-branch graph
+          if i_name not in nodes_outputs and i_name not in input_values.keys():
+            input_values[i_name] = kwargs["tensor_dict"][i_name]
```

Comment:
This block appears in many places. Is it possible to put it inside of the onnx_graph_to_tensorflow_ops API?

Reply:
@chinhuang007 Updated according to your suggestion, may you review it again, thanks.

Reply:
If you want to have a new copy of dictionary, I would suggest use deepcopy() instead of dict(..) which is shallow copy.